index.js

```diff
@@ -14,11 +14,10 @@ const moons = [
 
   You should not need to make any other changes to the code.
 */
-function organizeMoonsByPlanet(moons) {
+const organizeMoonsByPlanet = moons => {
   const result = {};
   for (let moon of moons) {
-    const name = moon.name;
-    const planet = moon.planet;
+    let {name:name, planet:planet} = moon;
```

Comment:
Although it doesn't break anything, the `:name` and `:planet` here are redundant. If you're changing the name, the `:` lets you do that.

```js
const person = {firstName: 'Colin', surName: 'Jaffe'};

const {firstName: first, lastName: last} = person;
```

This will give you variables called `first` and `last`. You're saying, "Save the values from the `firstName` and `surName` properties as variables, but name those variables `first` and `last`."

In your usage, you're saying, "Save the values from the `name` and `planet` properties as variables, but name those variables `name` and `planet`." But that's what they'd be called anyway!